Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -178,6 +177,12 @@ namespace Multiplayer
 
     void MultiplayerEditorConnection::OnDisconnect([[maybe_unused]] AzNetworking::IConnection* connection, [[maybe_unused]] DisconnectReason reason, [[maybe_unused]] TerminationEndpoint endpoint)
     {
-        ;
+        if (editorsv_isDedicated && m_networkEditorInterface->GetConnectionSet().GetConnectionCount() == 1)
+        {
+            if (m_networkEditorInterface->GetPort() != 0)
+            {
+                m_networkEditorInterface->StopListening();
```

Comment:
Should we stop listening MPNetworkInterfaceName instead? 
Otherwise if we do Ctrl+G again, the server won't get the connection from the Editor..

Reply:
Hmm yeah, that's possible if editorsv_launch is off. I'd mostly been thinking from the perspective of on. I can check that variable as well.

Reply:
Updated this to account for editorsv_launch so it's only closing the socket if the Editor launched the server for you.